config/rewrite/user.role.site_administrator.yml

```diff
@@ -1,20 +1,12 @@
 permissions:
   - 'access taxonomy overview'
-  - 'administer permissions'
```

Comment:
@pdscti Are these permissions provided by gla_core?

Reply:
The ones I've removed are, yes.